Assets/GothicVR/Scripts/Creator/MusicCreator.cs

```diff
@@ -50,6 +57,27 @@ void Start()
             musicSource = backgroundMusic.AddComponent<AudioSource>();
         }
 
+        /// <summary>
+        /// Called once every 0.20 seconds to update the music once the timer is bigger than the updateInterval
```

Comment:
Please add an Info "why" you do this. I assume it's because you want to have the music changed at a fixed time frame, right?